solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-9.adoc

```diff
@@ -347,4 +377,817 @@ where they really belong: /admin/threads, /admin/properties, /admin/logging
 
 * Atomic/partial updates to nested documents now _require_ the `\_root_` field to clearly show the document isn't a root document.  Solr 8 would fallback on the `\_route_` param but no longer.
 
-* SolrJ: The project is migrating away from the Apache HttpClient dependency in favor of Jetty's client (supporting HTTP/2), to occur over the 9x releases.  We deprecated HttpSolrClient and friends in favor of Http2SolrClient and equivalents.  For SolrCloud, the former CloudSolrClient was renamed CloudLegacySolrClient (deprecated), and instead BaseCloudSolrClient was renamed to CloudSolrClient and given a Builder for the Jetty based HTTP/2 client.
\ No newline at end of file
+* SolrJ: The project is migrating away from the Apache HttpClient dependency in favor of Jetty's client (supporting HTTP/2), to occur over the 9x releases.  We deprecated HttpSolrClient and friends in favor of Http2SolrClient and equivalents.  For SolrCloud, the former CloudSolrClient was renamed CloudLegacySolrClient (deprecated), and instead BaseCloudSolrClient was renamed to CloudSolrClient and given a Builder for the Jetty based HTTP/2 client.
+
+=== Solr Binary Release `dist/` Directory
+
+The `dist/` directory in the Solr binary release has been removed as of Solr 9.0.
+
+* The `solr-core` and `solr-solrj` jars can be found under `server/solr-webapp/webapp/WEB-INF/lib/`.
+* The Solr module jars can be found in `modules/<module-name>/lib`, packaged individually for each module.
+* The `solrj-deps` (SolrJ Dependencies) are no longer separated out from the other Server jars.
+Please refer to the SolrJ Maven artifact to see the exact dependencies you need to include from `server/solr-webapp/webapp/WEB-INF/lib/` and `server/lib/ext/` if you are loading in SolrJ manually.
+If you plan on using SolrJ as a JDBC driver, please refer to the xref:query-guide:sql-query.adoc#generic-clients[JDBC documentation]
+
+== Major Changes in Earlier 8.x Versions
+
+The following is a list of major changes released between Solr 8.1 and 8.11.
+
+Please be sure to review this list so you understand what may have changed between the version of Solr you are currently running and Solr 9.0.
+
+=== Solr 8.11
```

Comment:
Did you intend to add separate sections for all the 8x releases here or should we just have one consolidated section for 9.0?

Reply:
Separate sections for each 8.x release, just like we have separate sections for each 7.x here https://solr.apache.org/guide/8_0/major-changes-in-solr-8.html

Reply:
@janhoy I understand that this is the way we have been doing it for previous versions, but to me it makes very little sense. I think we should put the 8.x notes in major-changes-in-solr-8, and move the 7.x notes to major-changes-in-solr-7. I'm happy to do the work if others agree.

Reply:
It makes kind of sense, since the target audience for 9.0 would typically be someone upgrading from 8.x. And if you are on e.g. 8.4, you need to catch up on the diff between 8.4 - 8.11.

Could be that improvements can be made here, but let's not rush anything. I'd also check with Cassandra who I'm sure will have thougths on the subject as technical writer.